### DISCUSSION TITLE
How to change the ordinates of the population created in importing PK-sim from Mobi?

### DISCUSSION THREAD

**Discussion by balamo98** [Open-Systems-Pharmacology/Forum#1489] [Questions & Problems]
Hello，team，
I created a PD model from Mobi, and when I import it into PK-SIM to create a PD population, I want to change the ordinates and units，but I dont know how.
![图片](https://user-images.githubusercontent.com/96385989/228399525-fb9fa891-fb69-4750-a709-f6743b80c532.png)
I need to change its unit to "%".

thanks，yours colin

**Comment by StephanSchaller** [comment]
You have to change this unit first in MoBi. Cannot be changed in PKSim
